_posts/2016-10-26-jwt-artsy-journey.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: "JSON Web Tokens: Artsy's Journey"
+date: 2016-10-26 11:03
+comments: true
+author: ashkan
+categories: [Authentication, JWT, API]
+---
+
+At Artsy we currently have ~1730 client applications hitting our API and requesting authentication. When a user successfully authenticates through one of these clients, we want to embed basic user/application data in the resulting token rather than have to look up a session ID in the database on each request. For that we want to use JWT.
```

Comment:
Who is _we_ and why do you want to look up a session ID? I don't think from the mobile side we do that? I'm not even sure what a session ID refers to? unless it's the user auth representation?